src/symbol/mod.rs

```diff
@@ -1713,6 +1717,48 @@ impl<'t> TryFromCtx<'t, SymbolKind> for EnvBlockSymbol <'t> {
     }
 }
 
+/// A COFF section in a PE executable.
+///
+/// Symbol kind `S_SECTION`.
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct SectionSymbol<'t> {
+    /// Section number.
+    pub isec: u16,
+    ///  Alignment of this section (power of 2).
+    pub align: u8,
+    /// Reserved.  Must be zero.
+    pub reserved: u8,
+    /// Section's RVA.
+    pub rva: u32,
+    /// Section's CB.
+    pub cb: u32,
+    /// Section characteristics.
+    pub characteristics: u32,
```

Comment:
I think this can be [pe::SectionCharacteristics](https://github.com/afranchuk/pdb/blob/99259728aacdeb525e6b524cdf65f117e1226dc7/src/pe.rs#L102)?

Reply:
Most likely, the original structure is [`SECTIONSYM`](https://github.com/microsoft/microsoft-pdb/blob/805655a28bd8198004be2ac27e6e0290121a5e89/include/cvinfo.h#L4432)